src/Features/CSharp/Portable/Completion/CompletionProviders/EnumAndCompletionListTagCompletionProvider.cs

```diff
@@ -58,6 +58,14 @@ internal override bool IsInsertionTrigger(SourceText text, int characterPosition
                     return;
                 }
 
+                // Don't show up within member access
+                // This previously worked because the type inferrer didn't work
+                // in member access expressions.
+                if (token.IsKind(SyntaxKind.DotToken))
```

Comment:
I'm confused by this.  IsInsertionTrigger does not list ```.``` as a character.  So how come we're coming on on dot?

Thanks!